CONTRIBUTING.md

```diff
@@ -13,14 +13,20 @@
 │   ├── common
 │   ├── pages
 ├── img                                All the images go here
-│   ├── logos
-│   ├── pages
+│   ├── discord-page
+|   ├── icons
+|   ├── index-page
+|   ├── logos
+│   ├── shapes
```

Comment:
I feel mentioning every folder is redundant, since we can simple list the folder in the directory itself.

I would change it to something like:

```
|-- img                                All the images go here
|    |-- discord-page
|    |-- icons
|    ...
```

Reply:
ok

Reply:
New changes have been made kindly please check